test/gtest/Makefile.am

```diff
@@ -27,8 +27,13 @@ export UCX_HANDLE_ERRORS
 export UCX_LOG_LEVEL
 export UCX_LOG_PRINT_ENABLE
 
-export LSAN_OPTIONS=suppressions=$(top_srcdir)/contrib/lsan.supp
+if HAVE_ASAN
+export LSAN_OPTIONS=suppressions=$(abs_top_srcdir)/contrib/lsan.supp
```

Comment:
can it be outside if HAVE_ASAN?

Reply:
This one can, but `LD_PRELOAD` cannot so I decided to move all the ASAN related settings under if